src/frontend/src/hooks/auth.hooks.ts

```diff
@@ -16,16 +16,24 @@ export const useProvideAuth = () => {
   const [user, setUser] = useState<AuthenticatedUser | undefined>(undefined);
 
   const devSignin = async (userId: number) => {
-    const user = await mutateAsyncDev(userId);
-    setUser(user);
-    localStorage.setItem('devUserId', userId.toString());
-    return user;
+    try {
+      const user = await mutateAsyncDev(userId);
+      setUser(user);
+      localStorage.setItem('devUserId', userId.toString());
+      return user;
+    } catch (e) {
+      // do nothing; toasting error message will cause errors
+    }
   };
 
   const signin = async (id_token: string) => {
-    const user = await mutateAsync(id_token);
-    setUser(user);
-    return user;
+    try {
+      const user = await mutateAsync(id_token);
+      setUser(user);
+      return user;
+    } catch (e) {
+      // do nothing; toasting error message will cause errors
```

Comment:
this is the only file where toasting will actually cause an error, you should be able to do it in the other two places

Reply:
so for this file just don't do the try, catch since we can't toast an error